src/app/clusters/pump-configuration-and-control-server/pump-configuration-and-control-server.cpp

```diff
@@ -18,19 +18,338 @@
 #include <app/util/af.h>
 
 #include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/cluster-enums.h>
 #include <app-common/zap-generated/ids/Attributes.h>
+#include <app/AttributeAccessInterface.h>
+//#include <app/CommandHandler.h>
 #include <app/ConcreteAttributePath.h>
+#include <app/ConcreteCommandPath.h>
 #include <app/util/af-event.h>
 #include <app/util/attribute-storage.h>
 
 using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::PumpConfigurationAndControl;
-using namespace chip::app::Clusters::PumpConfigurationAndControl::Attributes;
+
+namespace {
+
+class PumpConfigurationAndControlAttrAccess : public AttributeAccessInterface
```

Comment:
This is the starting point of implementations that we are working on. It might be removed in future implementations if this shows to be the wrong method of handling read and write attribute 'overloading'.